fix(scim): update troubleshooting blurb about import users (#3924)
getsentry/sentry-docs: src/docs/product/accounts/sso/okta-sso/okta-scim.mdx

You can use "Push Groups" to sync and assign groups in Okta; they will be reflected in Sentry teams.

### Known Issues / Troubleshooting

> - Sentry does not currently support setting any User attributes other than `userName` and `active`.
> - Sentry does not support the Import Users feature fully.
>    - This is because the User `name` attribute will contain fields firstName and lastName with the values of `N/A`.
>    - Sentry's SCIM API does not currently support these fields but returns them for compatibility purposes.

Reviewer: ```suggestion
- Sentry does not currently support setting any User attributes other than `userName` and `active`.
- The Import Users feature is not currently supported. Sentry's SCIM API does not at this time support the User `name` attribute fields `firstName` and `lastName`. Instead, we return these with values of `N/A` for compatibility purposes.
```
